framework/src/org/checkerframework/common/value/ReflectiveEvaluator.java

```diff
@@ -94,6 +119,43 @@ public ReflectiveEvalutator(
     }
 
     /**
+     * This method changes a set of varargs invocations (each of which passes an arbitrary number of
```

Comment:
With regard to "invocation", you can change that too.  Varargs is fundamentally a declaration-site concept, so at first I thought that this was doing something with parameter lists.  It's not:  it is only changing argument lists.  Maybe you can find a better way to express that.